toolset/benchmark/test_types/framework_test_type.py

```diff
@@ -129,3 +134,89 @@ def copy(self):
         Use before calling parse
         '''
         return copy.copy(self)
+
+    def get_current_world_table(self):
```

Comment:
This is removed in #4549. We don't want any database specific stuff in the toolset. They should be contained to the `databases` folder.